sdks/python/apache_beam/dataframe/frames.py

```diff
@@ -922,33 +923,79 @@ def dropna(self, **kwargs):
   to_string = frame_base.wont_implement_method(
       pd.Series, 'to_string', reason="non-deferred-result")
 
-  def aggregate(self, func, axis=0, *args, **kwargs):
+  @frame_base.args_to_kwargs(pd.Series)
+  @frame_base.populate_defaults(pd.Series)
+  def aggregate(self, func, axis, *args, **kwargs):
```

Comment:
Should axis be 0 by default?

Reply:
That's actually what `frame_base.populate_defaults` does. It creates a wrapper method that has the same arguments, but with defaults populated from the equivalent pandas method. So this will end up with `axis=0` as the default

Reply:
Cool, out of curiosity, why have this be a decorator instead of having the defaults be in the argument list?

Reply:
Doing it programmatically keeps the pandas implementation as the single source of truth for these defaults, so we don't need to worry about inadvertently diverging (either because pandas changes them, or we mistakenly set the wrong default).

@robertwb actually added this so he may have some additional insight.

Reply:
Yep, that's exactly right. It both makes our intent clear here and guarantees the defaults line up.